app/Transformers/FileSetTransformer.php

```diff
@@ -152,7 +152,7 @@ public function transformForV4($fileset)
                     'chapter_end'   => $fileset->chapter_end,
                     'verse_start'   => $fileset->verse_start,
                     'verse_end'     => $fileset->verse_end,
-                    'thumbnail'     => $fileset->thumbnail,
+                    'thumbnail'     => $this->route === 'v4_filesets.chapter' ? $fileset->thumbnail : null,
```

Comment:
at first glance, this looks more restrictive.  how does this ensure the thumbnail is shown when it wasn't previously?